internal/http/services/owncloud/ocs/handlers/apps/sharing/shares/shares.go

```diff
@@ -702,11 +702,12 @@ func (h *Handler) listSharesWithMe(w http.ResponseWriter, r *http.Request) {
 
 		if data.State == ocsStateAccepted {
 			// Needed because received shares can be jailed in a folder in the users home
-			data.FileTarget = path.Join(h.sharePrefix, path.Base(info.Path))
-			data.Path = path.Join(h.sharePrefix, path.Base(info.Path))
+			data.FileTarget = path.Join(h.sharePrefix, info.Path)
+			data.Path = path.Join(h.sharePrefix, info.Path)
 		}
 
 		shares = append(shares, data)
+		log.Debug().Msgf("listSharesWithMe: %+v", *data)
```

Comment:
Remove the log

Reply:
It's a separate log for each public link, and we might have it duplicated in other places. So I think we can log `shares` at the end instead of individual entries

Reply:
Loging shares at the end won't help as the array is made of pointers and `%+v` will show the list of array pointers instead of the values. So either we keep it inside the loop or we confirm we log the shares in another place